theme/gnome-theme.css

```diff
@@ -20,6 +20,9 @@
 
 @import "parts/remove-white-flash.css";
 
+/* Extensions support */
+@import "parts/tab-center-reborn.css";
```

Comment:
Sorry, last change. 

I think better we should move this to `extensions/tab-center-reborn.css` and reserve `parts` for core things.